open-api-spec.yaml

```diff
@@ -823,15 +1013,8 @@ components:
           $ref: '#/components/schemas/amount'
         sendAmount:
           $ref: '#/components/schemas/amount'
-        fundsAvailable:
-          type: boolean
-          description: Indicates that there are funds available in the underlying account to complete this payment. This is not a guarantee they will still be available when the payment is attempted but it is a good indicator to a client that it should attempt to authorize the payment.
-          readOnly: true
-        expiresAt:
-          type: string
-          description: The date and time when the calculated `sendAmount` is no longer valid.
-          example: '2022-04-12T23:20:50.52Z'
-          readOnly: true
```

Comment:
If creating an outgoing payment:
- required a `quoteId` of a created quote and/or 
- synchronously performed the quote before resolving the create outgoing payment request

the outgoing payment schema could be a little cleaner:
- `receivingPayment`, `sendAmount`, `receiveAmount` would all be required
- no `receivingAmount`

Reply:
Otherwise, I'm considering implementing separate quoted and un-quoted outgoing payments database tables (maybe one [inheriting](https://www.postgresql.org/docs/current/ddl-inherit.html) from the other).

Reply:
https://github.com/interledger/open-payments/pull/97/commits/4bb20e6ebdb196b36b77d76dc6609eb7f9175c36 assumes that a request to create an outgoing payment without a pre-existing quote creates a quote synchronously.

Reply:
Or we would it be preferable to quickly return `202` + `Location` header for a request to create an outgoing payment without specifying a `quoteId`?

Reply:
https://github.com/interledger/open-payments/pull/97/commits/b8eec71fa60d56e24dd416821a3a279cf42a2d70 makes it so that a created `quoteId` is required when creating an outgoing payment.